packages/core/content-releases/admin/src/pages/tests/ReleaseDetailsPage.test.tsx

```diff
@@ -175,7 +175,9 @@ describe('Releases details page', () => {
     const moreButton = screen.getByRole('button', { name: 'Release actions' });
     expect(moreButton).toBeInTheDocument();
 
-    await user.click(moreButton);
+    await act(async () => {
+      await user.click(moreButton);
+    });
```

Comment:
I'm not sure this is the correct solution – would it make more sense to pass `skipHover` to the render options? So the user-event click callback doesn't try to hover the button which I assume is causing the tooltip to appear 🤔

Reply:
Didn't know about this 😅 , thank you, looks like it's working great